reporter-web-app/src/utils/index.js

```diff
@@ -40,4 +40,11 @@ const randomStringGenerator = (length = (Math.floor(Math.random() * 501) + 20))
     return alpha(l, symbol(1, alpha(r, []))).join('');
 };
 
-export { randomStringGenerator };
+// Utility function to generate color for given license string 
+const licenseToHslColor = (license) => {
+    return 'hsl(' + (Math.random() * (360 - 22) + 22) + ',' +
+        (25 + 70 * Math.random()) + '%,' + 
+        (55 + 10 * Math.random()) + '%)';
```

Comment:
I don't see how "license" is being taken into account here, is it? Because the same license should generate the same color, or?